.github/actions/setup-edge-linux/setup-edge.ps1

```diff
@@ -0,0 +1,13 @@
+function Program {
+    param(
+        [string]
+        $EdgeVersion
+    )
+
+    bash -c "curl -O https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-stable/microsoft-edge-stable_${EdgeVersion}_amd64.deb"
+    bash -c "sudo apt install ./microsoft-edge-stable_${EdgeVersion}_amd64.deb"
+
+    echo "/opt/microsoft/msedge" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
```

Comment:
This is awesome, though it would be nicer to have Edge installed on Linux by default. Please create an issue for this, reference https://github.com/actions/virtual-environments/issues/99 in it, and link it here.